regex-syntax/src/lib.rs

```diff
@@ -118,6 +133,9 @@ mod parser;
 mod unicode;
 mod unicode_tables;
 
+#[cfg(all(feature = "alloc", not(feature = "std")))]
+use alloc::string::String;
```

Comment:
It's a shame `alloc` doesn't have a prelude. Can I suggest making your own (containing the usual members only `std` isn't used) then doing `use ::prelude::*;` in all modules?

Reply:
Good idea! I'll see how this shakes out.